05_stringer/willshen8/stringer_test.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+	main()
+	expected := "127.0.0.1\n"
+	actual := buf.String()
+
+	assert.Equal(t, expected, actual, "Test Failed in main! Actual is not equal to expected result")
+}
+
+func TestStringer(t *testing.T) {
+	tests := []struct {
```

Comment:
I prefer test variables to be global vars

Reply:
I agree, as these can be reused if necessary.